marm-mcp-server/marm_mcp_server/core/write_queue.py

```
@@ -51,20 +61,34 @@ async def put(
-                self._resolve(request.future, memory_id)
+                if isinstance(request, MemoryWriteRequest):
+                    result = await self.memory.store_memory(
+                        request.content,
+                        request.session,
+                        request.context_type,
+                        request.metadata,
+                    )
+                else:
+                    result = await request.func(*request.args, **request.kwargs)
+                self._resolve(request.future, result)
             except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)